### PR TITLE
Updated polish translations

### PR DESCRIPTION
**Pull request by jezikk82**

### CODE REVIEW

File: app/src/main/res/values-pl/strings.xml [hunk 1]
```diff
@@ -1,203 +1,268 @@
...
+    <string name="MUSIC_TURN_REPEAT_ONE_ON">Powtarzaj pojedynczo </string>
+    <string name="MUSIC_TURN_REPEAT_OFF">Wyłącz powtarzanie</string>
+    <string name="MUSIC_SPOTIFY_REMOVE_FROM_COLLECTION">Usuń z kolekcji</string>
+    <string name="MUSIC_SPOTIFY_START_RADIO">Radio</string>
+    <string name="MUSIC_SPOTIFY_ADD_TO_COLLECTION">Polub</string>
+    <string name="MUSIC_SPOTIFY_THUMB_UP">Kciuk w górę</string>
+    <string name="MUSIC_SPOTIFY_THUMBS_UP_SELECTED">Kciuk w górę</string>
+    <string name="MUSIC_SPOTIFY_THUMB_DOWN">Kciuk w dół</string>
+    <string name="MUSIC_SPOTIFY_THUMBS_DOWN_SELECTED">Kciuk w dół</string>
+    <string name="lbl_cds_data" />
```
Comment (hufman) [inline app/src/main/res/values-pl/strings.xml:203]: This will hide this heading, can you add a translation instead?

Comment (jezikk82) [reply app/src/main/res/values-pl/strings.xml:203]: lbl_cds_data - this is the experimental tab when I was playing with extra tab to present cds data. I removed it through translation manager, but somehow I was left empty in strings.xml file. You can ignore it.

Comment (jezikk82) [reply app/src/main/res/values-pl/strings.xml:203]: It doesn't interfere with the main app.

**Comment by hufman** [comment]
Thank you so much!!

**Pull request merged**